mifospay/src/main/res/layout/toolbar.xml

```diff
@@ -8,4 +8,10 @@
     app:contentInsetRight="0dp"
     app:contentInsetEnd="0dp"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content" />
+    android:layout_height="wrap_content">
+    <ImageView
+        android:layout_width="@dimen/value_50dp"
+        android:layout_height="50dp"
```

Comment:
Here also use the dynamic value from `dimen.xml` file which you are using for width. Please avoid hardcoded values
and also don't forget to squash your commits into one.

Reply:
Thank you